src/gui/Window.qml

```diff
@@ -64,6 +67,11 @@ ApplicationWindow {
         )
     }
 
+    function drawAttention() {
+        window.show()
+        window.raise()
+        window.requestActivate()
```

Comment:
`drawAttention` is a confusing name, `restoreFromTray` would be more fitting (as oppose to "minimize to tray").
Are all three `window.` lines really needed? `requestActivate()` works directly on my desktop, not sure if this is the case elsewhere.

Reply:
Maybe different DEs/WMs have different behavior on each call. On KDE Plasma, without `show()`, it will not bring up anything. `raise()` in here brings up the window on top of everything. Without `raise()`, Mirage may show up under other windows. I believe `requestActivate()` tells WM a window requires attention. So I guess these 3 calls are necessary.